spec/index.html

```diff
@@ -89,9 +89,14 @@
   <body>
     <section id="abstract">
       <h2>Abstract</h2>
-<p>This document describes the use of HTTP operations for the purpose of managing a collection of RDF graphs. This interface is an alternative to the SPARQL 1.1 Update protocol. Most of the
-    operations defined here can be performed using that interface, but for some clients or servers, this interface may be easier to implement or work with. This specification may serve as a
-    non-normative suggestion for HTTP operations on RDF graphs which are managed outside of a SPARQL 1.1 graph store.</p>
+      <p>
+        This document describes the use of HTTP operations for the purpose of managing a
+        collection of RDF graphs.  This interface is an alternative to        [[[SPARQL12-PROTOCOL]]]. Most of the operations defined here can be performed
```

Comment:
```suggestion
        collection of RDF graphs.  This interface is an alternative to
        [[[SPARQL12-PROTOCOL]]]. Most of the operations defined here can be performed
```

Reply:
Noted - this will get done in the next PR cycle.